python/ray/llm/_internal/serve/deployments/utils/downloader_utils.py

```diff
@@ -106,12 +100,6 @@ class S3Downloader(BaseDownloader):
     def __init__(self, model_id: str, mirror_config: "S3MirrorConfig"):
         super().__init__(model_id, mirror_config)
 
-    def _get_env_vars(self):
-        env_vars = None
```

Comment:
the way we were managing aws creds was super hacky. Overriding env vars should be the way to go.